public/app/plugins/panel/graph/axes_editor.html

```diff
@@ -28,8 +29,20 @@ <h5 class="section-heading" ng-show="$index === 1">Right Y</h5>
 					<label class="gf-form-label width-6">Y-Max</label>
 					<input type="text" class="gf-form-input width-5" placeholder="auto" empty-to-null ng-model="yaxis.max" ng-change="ctrl.render()" ng-model-onblur>
 				</div>
-      </div>
-      <div class="gf-form">
+			</div>
+
+			<div class="gf-form-inline" ng-show="$index === 1">
```

Comment:
Great job.

However, after testing this I realized that it's a bit weird to have the Y-axis alignment under Right Y settings. I would suggest to remove Y-axis alignment from Right Y settings and create a new section Y-Axis below the X-Axis section. The new section will only be visible when both left and right Y are shown.

![image](https://user-images.githubusercontent.com/1668778/37661260-b275fd2e-2c54-11e8-8590-bf3bcd1802d8.png)

Above screenshot created using the following markup:
```html
<div class="section gf-form-group">
  <h5 class="section-heading">X-Axis</h5>
  ...  
  <div ng-if="ctrl.panel.yaxes[0].show && ctrl.panel.yaxes[1].show">
    <br/>
    <h5 class="section-heading">Y-Axis</h5>
    <gf-form-switch class="gf-form" label="Align" tooltip="Align left and right Y-axes" label-class="width-6" switch-class="width-5" checked="ctrl.panel.yaxis.alignment" on-change="ctrl.render()"></gf-form-switch>
    <div class="gf-form" ng-if="ctrl.panel.yaxis.alignment">
      <label class="gf-form-label width-6">Value</label>
      <input type="number" class="gf-form-input max-width-8" ng-model="ctrl.panel.yaxis.alignmentValue" placeholder="0" ng-change="ctrl.render()" ng-model-onblur bs-tooltip="'Align left and right Y-axes based on this value'" data-placement="right">
    </div>
  </div>
</div>
```

Your welcome to implement these changes if you want. If not, let me know and I will do the fixes before merging.

Reply:
Yes, this is a good improvement to enhance the user experience.

Reply:
The new section will be visible always, because aligned graphs on the axes, rather than the axes themselves. That is, the axes may not be visible, and the graphs are leveled.

Reply:
Could you please elaborate on that last comment. Why is there a need to display Y-Axis with alignment setting if you only have a left Y-axes shown?

Reply:
Align the graphics at the specified level. To align the graphs, the presence (display) of the axes is not necessary.
In the current stable version 5.0, if you unpress the show of Y-axes, the maximum and minimum values for the graphs will continue to be executed. However, the limits settings are hidden, I fixed it the same way. In order not to repeat this error, the level settings should always be displayed. But they will only be executed if the graphics are tied in both axes.

Reply:
Gotcha. You're right :+1: